tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Backward compatibility and test vector generation
+
+Crate `prio` supports Prio v2, used in [Exposure Notification Private Analytics](enpa-whitepaper), as implemented in [`prio-server`](https://github.com/abetterinternet/prio-server), as well as the Prio v3 Verifiable Distributed Aggregation Function used in [Privacy Preserving Measurements](ppm-spec).
+
+To ensure backward compatibility with older versions of crate `prio` and [other implementations of Prio v2](https://github.com/google/libprio-cc), we continuously test `prio`'s Prio v2 implementation against a test vector recorded with `prio` 0.4.0. That test is implemented in `tests/backward_compatibility.rs`. The test vector was generated using `tests/accumulating.rs`. To generate a new set of test vectors, run:
+
+    cargo test --test accumulating -- --nocapture
```

Comment:
What about creating a binary for generating test vectors? Maybe call it `prio_v2_test_vector_gen`?

Reply:
You caught me, I was getting lazy here. I'll add a binary target to the crate and do this properly. I bet we'll want to generate lots of more test vectors in the future so having a more robust tool to build on for this will probably pay off. Plus, we'll end up with a `Cargo.lock` in `libprio-rs` so that we can go back and rebuild old versions of the crate even if they use yanked dependencies.